.github/workflows/license-generation.yml

```diff
@@ -95,9 +106,11 @@ jobs:
 
       - uses: actions/setup-node@v2
```

Comment:
should we move some of the actions themselves to the latest version (`v4` I think here) 😋